src/app/modules/contact/contact.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="contact-container">
+    <div class="contact">
+        {{"📬Contact with us"}}
```

Comment:
{{ }} to się nazywa string interpolation i https://v17.angular.io/guide/interpolation. Używamy tych znaczków jeśli w pliku ts miałbyś np zmienną contactUsText: string = 'Contact with us' a tak to wystarczy bez tych {{ }}